src/Reflection/ReflectionEnumCase.php

```diff
@@ -22,13 +23,38 @@
 
 class ReflectionEnumCase
 {
+    private string $name;
+
+    private Node\Expr|null $value;
+
+    /** @var list<ReflectionAttribute> */
+    private array $attributes;
+
+    private string $docComment;
+
+    private int $startLine;
+
+    private int $endLine;
+
+    private int $startColumn;
+
+    private int $endColumn;
```

Comment:
We should declare stricter types like `positive-int` or `non-empty-string` for these

Reply:
Done.